### PR TITLE
Fix linter

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter warnings

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -4,10 +4,10 @@
     "packaging_format": 1,
     "license": "LGPL-2.1,GPL-2.0",
     "url": "https://sogo.nu/",
-    "version": "4.0.7-1~ynh2",
+    "version": "5.0.1~ynh1",
```
Comment (Josue-T) [inline manifest.json:7]: Well we use the debian package from debian so it still be the version 4.0.7 on debian buster.

File: scripts/restore [hunk 2]
```diff
@@ -18,12 +18,8 @@ ynh_script_progression --message="Loading settings..."
 
 domain=$(ynh_app_setting_get --app $app --key domain)
 path_url=$(ynh_app_setting_get --app $app --key path)
-final_path=$(ynh_app_setting_get --app $app --key final_path)
 db_name=$(ynh_app_setting_get --app $app --key db_name)
-
-# CHECK IF THE APP CAN BE RESTORED
```
Comment (Josue-T) [inline scripts/restore:24]: Why do you remove this ?

**Review by Josue-T — CHANGES_REQUESTED** [review]
It looks like ok, we just still need to fix this.

File: manifest.json (hunk 1 above)
Comment (ericgaspar) [reply manifest.json:7]: ah, this is because I am using Bullseye...

File: scripts/restore (hunk 2 above)
Comment (ericgaspar) [reply scripts/restore:24]: `ynh_webpath_available` is no longer needed, it is now handled by the core

File: manifest.json (hunk 1 above)
Comment (Josue-T) [reply manifest.json:7]: Yes but as I know actually we still be on buster on the stable version...

**Comment by Josue-T** [comment]
!testme

**Pull request closed without merging**